### PR TITLE
Introduce CRYSTAL price fluctuation MVP

### PR DESCRIPTION
**Pull request by ipdae**
- MVP for CRYSTAL price.
- Calculate the crystal cost by the ratio of the previous week's and before previous week's crystal costs.
- Add cost history by day & week.

### CODE REVIEW

File: Lib9c/Helper/CrystalCalculator.cs [hunk 1]
```diff
@@ -2,10 +2,13 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Numerics;
+using Libplanet;
 using Libplanet.Assets;
 using Nekoyume.Model.Item;
+using Nekoyume.Model.State;
 using Nekoyume.TableData;
 using Nekoyume.TableData.Crystal;
+using Serilog;
```
Comment (sonohoshi) [inline Lib9c/Helper/CrystalCalculator.cs:11]: This namespace need?

File: Lib9c/Addresses.cs [hunk 2]
```diff
@@ -30,5 +32,15 @@ public static class Addresses
         public static Address GetSheetAddress(string sheetName) => TableSheet.Derive(sheetName);
 
         public static Address GetItemAddress(Guid itemId) => Blacksmith.Derive(itemId.ToString());
+
+        public static Address GetDailyCrystalCostAddress(int index)
+        {
+            return MaterialCost.Derive($"daily_{index.ToString(CultureInfo.InvariantCulture)}");
```
Comment (sonohoshi) [inline Lib9c/Addresses.cs:38]:
```suggestion
            return MaterialCost.Derive($"daily_{index.ToString()}");
```
I think... there could be problems depending on the user's culture setting if CultureInfo goes in.

**Review by Unengine — APPROVED** [review]

File: Lib9c/Addresses.cs (hunk 2 above)
Comment (ipdae) [reply Lib9c/Addresses.cs:38]: I used InvariantCultureInfo to be unaffected by the user's environment. https://docs.microsoft.com/ko-kr/dotnet/api/system.globalization.cultureinfo.invariantculture?view=net-6.0

Comment (sonohoshi) [reply Lib9c/Addresses.cs:38]: I misunderstood. Thank you for your answer!

**Review by sonohoshi — APPROVED** [review]

**Pull request merged**